avaframe/com4FlowPy/flowCore.py

```diff
@@ -327,10 +335,14 @@ def calculation(args):
     cellsize = args[8]
     infraBool = args[9]
     forestBool = args[10]
+    uMaxBool = args[11]
+    uMax = args[12]
```

Comment:
maybe use ```uMaxArray``` instead of ```uMax``` to indicate that the variable is an array and not a scalar value